src/lib/utils/application-commands/getNeededParameters.ts

```diff
@@ -18,6 +18,7 @@ export async function getNeededRegistryParameters() {
 async function fetchGuildCommands(commands: ApplicationCommandManager) {
 	const map = new Map<string, Collection<string, ApplicationCommand>>();
 
+	// TODO(vladfrangu): v3/v4 improve this with promise.all instead
```

Comment:
TODO

Reply:
As stated on discord, if you're not going to pick this up in this PR then greater a GH issue out of it and remove the to-do comment.

I don't want to see these kind of Todo comments in the code

- it's not clear for contributors
- they'll be forgotten
  - you stated that's the intent, in that case don't add them and don't create the issue.
- Todo comments in code are an anti-pattern. They're effectively useless.